CustomLogAppender/README.md

```diff
@@ -1,28 +1,26 @@
 # Custom Log Appender
-Demonstrates how you could write a Log Appender that can be enabled to be used by Axon Ivy. trough the `configurations/log4jconfig.xml` file.
+Demonstrates how you could write a Log Appender that can be enabled to be used by Axon Ivy.
+trough the `configurations/log4j2.xml` file. The appender attaches itself to desired logs.
+To see an output you need an error log in the project to which the appender can attach itself.
```

Comment:
make an example, where must this logger be exactly? java, ivyscript? can you give me an example log line, that would trigger the appender we have demonstrated?